blog/views.py

```diff
@@ -1,4 +1,7 @@
 """Views for blog app."""
 from django.shortcuts import render  # noqa: F401
 
-# Create your views here.
+
+def post_list(request):
+    """Render requested page with list of posts."""
+    return render(request, 'blog/post_list.html', {})
```

Comment:
This PR would be non-atomic without this template in place. Could you please add it as well?